src/v2/guide/list.md

```diff
@@ -440,6 +440,10 @@ De forma similar a la plantilla `v-if`, también puede usar una etiqueta `<templ
 
 ## `v-for` con `v-if`
 
+nota: Tenga en cuenta que no se recomienda usar v-if y v-for juntos. Consulte la [guía de estilo](https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential) para más detalles.

```

Comment:
Hey @carlos00027, thanks for noticing this new content! Please, can you update this PR to include the HTML and markup?

```suggestion
<p class="tip">Tenga en cuenta que **no se recomienda** usar `v-if` y `v-for` juntos. Consulte la [guía de estilo](/v2/style-guide/#Avoid-v-if-with-v-for-essential) para más detalles.</p>
```